src/test/storage/adaptive_radix_tree_index_test.cpp

```diff
@@ -43,6 +44,40 @@ class AdaptiveRadixTreeIndexTest : public BaseTest {
       pairs.emplace_back(std::make_pair(bc, values1[i]));
     }
     root = index1->_bulk_insert(pairs);
+
+    std::random_device rd;
+    _rng = std::mt19937(rd());
+  }
+
+  void _search_elements(std::vector<int>& values) {
```

Comment:
More precise data types?

Reply:
Would make sense, but honestly I just extended the existing tests to cover more corner cases.
We should type these tests, but I doubt I find the time the upcoming weeks.